src/ceph-daemon/ceph-daemon

```diff
@@ -1644,7 +1799,12 @@ def command_adopt():
 
 def command_rm_daemon():
     # type: () -> None
-    (daemon_type, daemon_id) = args.name.split('.', 1)
+    if '.' in args.name:
+        (daemon_type, daemon_id) = args.name.split('.', 1)
+    else:
+        daemon_type = args.name
+        daemon_id = None
+
```

Comment:
sigh this is everywhere.. makes me liek teh idea of sticking to $type.$id even when we only expect one of them

Reply:
see prior comment about defaulting to 0 for the id if not present

Reply:
see above - latest commit uses hostname as the daemon_id

Reply:
I would just have the user pass ``--name prometheus.foo`` instead of silently add it--less surprising.  (And also mgr/ssh can then use whatever naming convention it likes)

Reply:
ACK